.github/workflows/core_deploy_weu.yaml

```diff
@@ -0,0 +1,164 @@
+name: WEU - Core TF Apply (PROD)
+
+# Only manual
+on: 
+  workflow_dispatch:
+    inputs:
+      prod_environment:
+        description: "Run on PROD environment"
+        type: choice
+        required: true
+        default: 'true'
+        options:
+          - true
+          - false
+      use_private_agent:
+        description: Use a private agent to run the Terraform plan.
+        type: boolean
+        required: false
+        default: false
```

Comment:
To check if the self-hosted runner is mandatory due to the existence of private resources